backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5776250_sys_gh26307_Business_Partner_Window.sql

```diff
@@ -0,0 +1,147 @@
+;
+
+-- 2025-11-10T18:04:03.718Z
+INSERT INTO AD_Field_Trl (AD_Language,AD_Field_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Field_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Field t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Field_ID=755958 AND NOT EXISTS (SELECT 1 FROM AD_Field_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Field_ID=t.AD_Field_ID)
+;
+
+-- 2025-11-10T18:04:03.784Z
+/* DDL */  select update_FieldTranslation_From_AD_Name_Element(584201)
+;
+
+-- 2025-11-10T18:04:03.853Z
+DELETE FROM AD_Element_Link WHERE AD_Field_ID=755958
+;
+
+-- 2025-11-10T18:04:03.919Z
+/* DDL */ select AD_Element_Link_Create_Missing_Field(755958)
+;
+
+-- UI Element: Geschäftspartner_OLD(123,D) -> Geschäftspartner(220,D) -> advanced edit -> 10 -> advanced edit.Erzeugernummer
+-- Column: C_BPartner.WasteProducerNo
+-- 2025-11-10T18:05:12.046Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_ElementGroup_ID,AD_UI_Element_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayedGrid,IsDisplayed_SideList,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNoGrid,SeqNo_SideList,Updated,UpdatedBy) VALUES (0,755958,0,220,540671,638573,'F',TO_TIMESTAMP('2025-11-10 18:05:11.447000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'Y','N','N','Y','N','N','N',0,'Erzeugernummer',170,0,0,TO_TIMESTAMP('2025-11-10 18:05:11.447000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
+
+-- UI Element: Geschäftspartner_OLD(123,D) -> Geschäftspartner(220,D) -> advanced edit -> 10 -> advanced edit.Ensorgernummer
+-- Column: C_BPartner.WasteDisposerNo
+-- 2025-11-10T18:05:48.316Z
+INSERT INTO AD_UI_Element (AD_Client_ID,AD_Field_ID,AD_Org_ID,AD_Tab_ID,AD_UI_ElementGroup_ID,AD_UI_Element_ID,AD_UI_ElementType,Created,CreatedBy,IsActive,IsAdvancedField,IsAllowFiltering,IsDisplayed,IsDisplayedGrid,IsDisplayed_SideList,IsMultiLine,MultiLine_LinesCount,Name,SeqNo,SeqNoGrid,SeqNo_SideList,Updated,UpdatedBy) VALUES (0,755957,0,220,540671,638574,'F',TO_TIMESTAMP('2025-11-10 18:05:47.780000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'Y','N','N','Y','N','N','N',0,'Ensorgernummer',180,0,0,TO_TIMESTAMP('2025-11-10 18:05:47.780000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
```

Comment:
please move these fields in the advanced edit (single view)  of the Customer and Vendor tabs.